projects/js-packages/components/components/dialog/style.module.scss

```diff
@@ -27,3 +20,11 @@
 		width: 100%;
 	}
 }
+
+// Viewport breakpoints.
+.is-viewport-small {
+	// Add spacing between primary and secondary in "sm" viewport size
+	.primary {
+		margin-bottom: calc( var( --spacing-base ) * 3 );
+	}
+}
```

Comment:
Just in terms of organization, is not better for `.is-viewport-small` to be inside `.primary`?

```css
.primary {
  .is-viewport-small {
  }
}
```